src/commands/BootstrapCommand.js

```diff
@@ -3,15 +3,40 @@ import NpmUtilities from "../NpmUtilities";
 import PackageUtilities from "../PackageUtilities";
 import Command from "../Command";
 import async from "async";
-import find from "lodash/find";
+import {find} from "lodash";
 import path from "path";
 import semver from "semver";
 
+const SUPPORTED_OPTS = {
+  npmClient: {
+    describe: "Executable used to install dependencies (npm, yarn, pnpm, ...)",
+    type: "string",
+    requiresArg: true
+  },
+  hoist: {
+    describe: "Install external dependencies matching [glob] to the repo root.  Use with no glob for all.",
+    type: "string"
+  },
+  nohoist: {
+    describe: "Don't hoist external dependencies matching [glob] to the repo root",
+    type: "string"
+  }
+};
+
 export default class BootstrapCommand extends Command {
+  static getSupportedOptions() {
+    return Object.assign({}, Command.getSupportedOptions(), SUPPORTED_OPTS);
+  }
+
+  static get describe() {
```

Comment:
As I've shown elsewhere, this should just be a named export, not a static property of the default export.